docs/markdown/navigation/guards_verification.md

```diff
@@ -0,0 +1,53 @@
+<!-- .slide: class="sfeir-basic-slide" -->
+# Fonctionnement des guards de "vérification"
+<br><br>
+<ul>
+    <li>Peut renvoyer un observable, une promesse ou encore un boolean</li><br>
+    <li>Un gards est un service classique</li><br>
+    <li>Si true alors on navigue vers l'url demandé</li><br>
+    <li>Si false on ne navigue pas vers l'url demandé</li><br>
+</ul>
+
+##==##
+
+<!-- .slide: class="sfeir-basic-slide with-code" -->
+# Un exemple plus parlant
+<br><br>
+```typescript
+@Injectable({
+  providedIn: CoreModule,
+})
+export class LoginGuards implements CanLoad {
+  constructor(private readonly authentificationService: AuthentificationService, private readonly router: Router) {}
+
+  canLoad(next: Route, segments: UrlSegment[]): Observable<boolean> | boolean {
+    return this.verifyToken();
+  }
+
+  verifyToken(): Observable<boolean> {
+    return this.authentificationService.verifyToken()
+      .pipe(
+        map(() => true),
+        catchError(() => {
+          this.router.navigate(['/login']);
+          return of(false);
+        }),
+      );
+  }
+}
+```
```

Comment:
this code bloc does not fit horizontally and vertically. This is not very practical